sova/git/rebase.py

```
@@ -24,6 +43,133 @@ class RebaseResult:
+
+    async def _invoke_model(model_id: str) -> tuple[str | None, Decimal]:
+        provider = providers[model_id]
+        template = _find_prompt_template(model_id, prompt_templates)
+        prompt = _build_file_resolution_prompt(filename, file_content, template=template)
+        try:
+            result = await provider.invoke(
+                prompt,
+                model=model_id,
+                max_budget_usd=per_model_budget,
+            )
+            return result.text, result.cost_usd
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)